specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -71,6 +79,17 @@ Non-goals are only features, which a contributor can reasonably assume were a go
 One example is features that were cut during scoping.
 -->
 
+### Goals
+
+* **Introduce a vdk-meta-jobs plugin**
+
+* **Provide a way to manage job dependencies**
+    - A user wants to execute jobs in a strict order. For example, one wants to ingest data from two different sources
```

Comment:
I think removing the user would be better. Just "execute jobs in a strict order" seems better to me, but I might be wrong.

Reply:
Makes sense, yes. Done.